activitysim/examples/example_mtc/configs/settings.yaml

```diff
@@ -184,6 +184,7 @@ models:
   - school_location
   - workplace_location
   - auto_ownership_simulate
+  - vehicle_type_choice
```

Comment:
I believe this is where you specify that this should be run for the MTC example - but when I ran `test_mtc.py` it did not run this set of steps. Is it inheriting from a different config? If so we should make sure `vehicle_type_choice`  is included in the test examples.

Reply:
Yes, see [here](https://github.com/ActivitySim/activitysim/blob/master/activitysim/examples/example_mtc/test/test_mtc.py#L45-L46). It first reads settings.yaml from `example_mtc/test/configs/`, and then appends these to the settings.yaml in `example_mtc/configs`. The `test_mtc.py` script also runs two other examples that _don't_ read from `example_mtc/test_configs`, however, (see [here](https://github.com/ActivitySim/activitysim/blob/master/activitysim/examples/example_mtc/test/test_mtc.py#L68-L69)), so maybe you just missed it when it ran?

Reply:
@mxndrwgrdnr - I have run the MTC example after installing activitysim from this PR using both:
```bash
python activitysim/examples/example_mtc/test/test_mtc.py
```
and 

```bash
pytest -s python activitysim/examples/example_mtc/test/test_mtc.py
```
but still couldn't find evidence of the vehicle choice model running. 

To see if it ran, I checked the log files and also grepped the entire activity sim directory for `vehicle_id`. 

I checked the test logs on Travis and there isn't a way to confirm that it ran from Travis - which tells me that we should probably have more feature-oriented testing capability.

Reply:
@mxndrwgrdnr does the code write out the generated vehicle table yet?